Misc/NEWS.d/next/Library/2020-10-03-23-14-50.bpo-41923.Buonw9.rst

```diff
@@ -0,0 +1 @@
+Implement :pep:`613`, introducing :class:`typing.TypeAlias` annotation.
```

Comment:
Could you mention that you just copied this code from typing_extensions? It doesn't seem fair to claim authorship for this code, as you imply here and in your change to Misc/ACKS. You only deserve credit for figuring out where to put the new code. :-)

Reply:
Touché!